docs/examples/pipelinerun_timeout.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: steward.sap.com/v1alpha1
+kind: PipelineRun
+metadata:
+  generateName: sleep
+spec:
+  timeout: "1d"
```

Comment:
`d` is not allowed by [`time.ParseDuration()`](https://pkg.go.dev/time#ParseDuration).

Reply:
This was committed by mistake. It was a check if CRD pattern works. Will be fixed.